lib/functoria/lib.ml

```diff
@@ -119,16 +122,18 @@ module Make (P : S) = struct
     let context = Key.context ~stage:`Configure ~with_required keys in
     let context = Context_cache.merge cache context in
     let f context =
-      let r, i = Key.eval context info context in
-      match output with None -> (r, i) | Some o -> (r, Info.with_output i o)
+      let config = Key.eval context info context in
+      match output with
+      | None -> config
+      | Some o -> { config with info = Info.with_output config.info o }
     in
     Cmdliner.Term.(pure f $ context)
 
   (* FIXME: describe init *)
   let describe (t : _ Cli.describe_args) =
-    let (_, jobs), _ = t.Cli.args.context in
+    let { Config.jobs; _ } = t.args.Cli.context in
     let f fmt =
-      Fmt.pf fmt "%a\n%!" (if t.dot then Config.pp_dot else Config.pp) jobs
+      Fmt.pf fmt "%a\n%!" (if t.dot then Config.pp_dot else Fmt.nop) jobs
```

Comment:
Is it possible to continue to show something sensible in text mode (instead of `Fmt.nop`) ?

Reply:
`Config.pp` was defined as `Device_graph.pp` ... which was `Fmt.nop`. It was printing nothing all along.

Reply:
We could technically use `Impl.pp` here, but that's more of a debug output than something you want to show to users.